src/main/java/aztech/modern_industrialization/attributes/QuantumArmorAttribute.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Azercoco & Technici4n
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package aztech.modern_industrialization.attributes;
+
+import aztech.modern_industrialization.MIText;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.world.entity.ai.attributes.AttributeModifier;
+import net.minecraft.world.item.TooltipFlag;
+
+public class QuantumArmorAttribute extends DisplayNamedAttribute {
+    public QuantumArmorAttribute() {
+        super(MIText.AttributeQuantumArmor.getTranslationKey(), 0);
+    }
+
+    @Override
+    public MutableComponent toValueComponent(AttributeModifier.Operation operation, double value, TooltipFlag flag) {
+        return Component.literal("\u00B9\u2044\u2084 |\u221E> + \u00B3\u2044\u2084 |0>");
```

Comment:
The hardcoded string only makes sense when dealing with items which grant the bonus of one piece of quantum armor. However, attribute viewers such as the one added by Apothic Attributes may render the quantum armor attribute with a value that is not exactly 1; the other common value being 4 when wearing a full set of Quantum Armor. The same issue can occur with custom items.

Reply:
Neoforge handles this for us, so not really an issue
![image](https://github.com/user-attachments/assets/ab5a6aec-4009-474d-939d-4eaaf87d0d5c)